docs/source/pages/developers_guide/index.rst

```diff
@@ -113,10 +136,25 @@ Feel free to explore other existing checks under ``src/macaron/slsa_analyzer/che
        git_repo: Mapped[str] = mapped_column(String, nullable=True, info={"justification": JustificationType.HREF})
 
        __mapper_args__ = {
-           "polymorphic_identity": "__repo_check",
+           "polymorphic_identity": "_repo_check",
        }
 
-3. Add a class for your check, provide the check details in the initializer method, and implement the logic of the check in ``run_check``.
+'''''''''''''''''''
+Add the check class
+'''''''''''''''''''
+
+Add a class for your check that subclasses :class:`BaseCheck <macaron.slsa_analyzer.checks.base_check.BaseCheck>`,
+provide the check details in the initializer method, and implement the logic of the check in
+:func:`run_check <macaron.slsa_analyzer.checks.base_check.BaseCheck.run_check>`.
+
+A ``check_id`` should meet the following requirements:
+
+    - The general format: ``mcn_<name>_<digits>``
+    - In ``name``, only lowercase alphabetical letters are allowed. If ``name`` contains multiple \
+        words, they must be separated by underscores.
```

Comment:
This currently renders like this on my side:
![image](https://github.com/oracle/macaron/assets/49882031/ec8953bc-5840-4de1-87f6-ad2529b70478)

I suspect this is due to the additional indentation on the second line relative to the first, so Sphinx interpreted this as a [definition list item](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#definition-lists). This is instead a [bullet list item](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#bullet-lists) item, so the second line should be indented at the same level as the first line. The backslash at the end of the first line is also not necessary.

```suggestion
    - In ``name``, only lowercase alphabetical letters are allowed. If ``name`` contains multiple
      words, they must be separated by underscores.
```

Reply:
> This currently renders like this on my side

See comit [4661c00](https://github.com/oracle/macaron/pull/620/commits/4661c001a72ce289b550fe11591e031f331f943c).